Redis enterprise (#2312)
DataDog/integrations-extras: redis_enterprise/README.md

# Agent Check: Redis Enterprise

## Overview

> [Redis Enterprise][1] integration provides all metrics for three critical aspects of a cluster; database, node, and shard. This enables fine-grained reporting that makes it easier to detect problems before they become critical. For a full list please see the 'Metrics' section.

Reviewer: ```suggestion
The [Redis Enterprise][1] integration provides metrics for three critical aspects of a cluster; database, node, and shard. This enables detailed reporting that makes it easier to detect problems before they become critical. For a full list, see the [metadata.csv][6].
```

## Setup

### Installation

> Run the following command to install the Agent integration:
> 
> 1. datadog-agent integration install -t datadog-redis_enterprise==1.0
> 
> 2. Configure the integration by setting the openmetrics_endpoint to your cluster's master node. See [Integration][2] for further information.
> 
> 3. [Restart][3] the agent.

Reviewer: ```suggestion
1. Run the following command to install the Agent integration:
    ```shell
    datadog-agent integration install -t datadog-redis_enterprise==1.0
    ```

2. Configure the integration by setting the `openmetrics_endpoint` to your cluster's master node. See [Integration][2] for further information.

3. [Restart][3] the agent.
```

### Configuration

> Set the openmetrics_endpoint to point to your cluster. See the [example][4].

Reviewer: ```suggestion
Set the `openmetrics_endpoint` to point to your cluster. See the [example][4].
```

### Validation

> 1. Ensure you can ping the machine, particularly in a cloud environment. Run wget or curl to ensure that you can receive metrics.

Reviewer: ```suggestion
1. Ensure you can ping the machine, particularly in a cloud environment. Run `wget` or `curl` to ensure that you can receive metrics.
```

# Agent Check: Redis Enterprise

## Overview

> [Redis Enterprise][1] Software is a self-managed data platform that unlocks the full potential of Redis at enterprise scale. The speed you know and love, plus compliance, reliability, and unmatched resiliency for enterprise needs.

Reviewer 2: The overview should include the below instead of Marketing copy: 
	1	What does your product do in 1-2 sentences
	2	What will customers get from this integration (the value of the integration & why it's helpful)
	3	What types of data sent and the value

Author: Marketing copy replaced, points 1-2 addressed, point 3 is answered in the Metrics section.

Author: This has been addressed

> The [Redis Enterprise][1] integration provides metrics for three critical aspects of a cluster; database, node, and shard. This integration enables fine-grained reporting that makes it easier to detect problems before they become critical. For a full list see the 'Metrics' section.

Reviewer 2: Could you provide 1-2 sentences on what your product does?

Reviewer 2: example - this is where you mention that this is on-prem etc so when you release the cloud version, customers/users can clearly tell the differences.

## Setup

### Installation

1. Run the following command to install the Agent integration:
   ```shell

>    datadog-agent integration install -t datadog-redis_enterprise==1.0

Reviewer 2: ```suggestion
   datadog-agent integration install -t datadog-redis_enterprise==1.0.0
```

(https://docs.datadoghq.com/developers/integrations/python/)
 on any machine.

2. Checkout the [repository][2] 

3. Run `ddev release build redis_enterprise` to build the package.

3. [Download the Datadog Agent][3].

4. Upload the build artifact to any host with an Agent and
 run `datadog-agent integration install -w

>  path/to/redis_enterprise/dist/<ARTIFACT_NAME>.whl`. For information regarding starting and stopping the agent, see [start, stop, and restart the agent][5]. For other information about the agent, see [agent status and information][6].

Reviewer 2: The steps should look similar to the steps in this [article](https://docs.datadoghq.com/agent/guide/use-community-integrations/?tab=agentv721v621#setup). 

Run the following command to install the Agent integration:

datadog-agent integration install -t datadog-<INTEGRATION_NAME>==<INTEGRATION_VERSION>
The version for the integration can be found in the respective changelog on the integration’s Github repository

Configure your integration similar to core [integrations](https://docs.datadoghq.com/getting_started/integrations/).

[Restart the Agent](https://docs.datadoghq.com/agent/configuration/agent-commands/#restart-the-agent).

Author: I pointed them at start, stop and restart the agent but can change it to simply restart if you prefer.

# Agent Check: Redis Enterprise

## Overview

The [Redis Enterprise][1] integration provides metrics for three critical aspects of a cluster; database, node, and shard. This integration enables fine-grained reporting that makes it easier to detect problems before they become critical. For a full list see the 'Metrics' section.

## Setup

### Installation

> Run the following command to install the Agent integration:

Reviewer: ```suggestion
```

Reviewer 2: @j8-redis bumping this
